core/store/migrate/migrations/0123_terra_idx_simplify.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+-- Simplify into one index to improve write performance.
+DROP INDEX IF EXISTS idx_terra_msgs_terra_chain_id_state;
+DROP INDEX IF EXISTS idx_terra_msgs_terra_chain_id_contract_id_state;
+-- We order by state first, then contract_id, to permit efficient queries when grouping unstarted txes
+-- across contracts.
```

Comment:
tradeoff being slower queries for getting messages in all states for a given contract_addr, but we don't currently have such a query (also seems unlikely to be needed)